app/environments/brassbirmingham/brassbirmingham/envs/test.py

```diff
@@ -186,24 +187,27 @@ def testVictoryPoints(self):
         
         p1CottonBuilding = p1.buildings[10]
         p1.buildBuilding(p1CottonBuilding, birmingham.buildLocations[0])
-        p1CottonBuilding.sell() # 5
+        # p1CottonBuilding.sell() # 5
+        # p1.sell(p1CottonBuilding) #can't sell an industry building
```

Comment:
`sell` shouldn't be a function of `Building` then, right? Only `MarketBuilding`

Reply:
Probably right, I suppose I made it on `Building` for when coal runs out, like it's 'sold' - flipped, same mechanic, but we can always rename and/or move it.